internal/api/api.go

```diff
@@ -148,6 +150,29 @@ type ContainerCatalogEntry struct {
 		TopLayerID             string `json:"top_layer_id"`
 		UncompressedTopLayerID string `json:"uncompressed_top_layer_id"`*/
 }
+type ChartStruct struct {
+	Entries map[string][]struct {
+		Name        string `json:"name"`
+		Version     string `json:"version"`
+		KubeVersion string `json:"kubeVersion"`
```

Comment:
Why not use KubeVersion to check that the platform can run this helm chart :
[deliedit@fedora newrepo]$ oc version
Client Version: 4.8.13
Server Version: 4.8.3
Kubernetes Version: *v1.21.1*+051ac4f